package.json

```diff
@@ -16,9 +16,9 @@
     "upgrade:tamagui": "yarn up '*tamagui*'@latest '@tamagui/*'@latest react-native-web-lite@latest",
     "upgrade:tamagui:canary": "yarn up '*tamagui*'@canary '@tamagui/*'@canary react-native-web-lite@canary",
     "check-deps": "check-dependency-version-consistency .",
-    "prettier": "prettier --ignore-path .gitignore '**/*.+(js|jsx|json|ts|tsx|md|yml|yaml)'",
-    "format": "yarn prettier --write",
-    "check-format": "yarn prettier --list-different",
+    "_prettier": "prettier --ignore-path .gitignore '**/*.+(js|jsx|json|ts|tsx|md|yml|yaml)'",
```

Comment:
It was a suggestion from expo doctor because the scripts was the same as the binary. So it'd override the behvaiour of `yarn prettier` (which would normally execute the binary)

Reply:
Ahh I did not know that `yarn prettier` picks the binary. Is this a v3 thing? I do think that `_prettier` will be forgotten to be used and `yarn prettier` will be ran. But then again it likely `yarn format` and `yarn check-format` will be used most.

Reply:
> Ahh I did not know that yarn prettier picks the binary. Is this a v3 thing?

No I think that's a yarn thing in general.